src/Transcript-Core/Object.extension.st

```diff
@@ -29,31 +30,33 @@ Object >> logExit [
 
 { #category : #'*Transcript-Core' }
 Object >> trace [
-	self trace: self printString
+	self trace: self
 ]
 
 { #category : #'*Transcript-Core' }
 Object >> trace: aString [
 	"Log the argument. Use self trace: instead of Transcript show: "
 	
-	Transcript show: aString
+	Transcript show: aString asString
```

Comment:
the argument should be called anObject instead of aString since you are sending #asString to it

Reply:
Yes! Thanks for the feedback